src/Compilers/Core/Portable/PEWriter/PeWriter.cs

```diff
@@ -104,6 +104,7 @@ internal static class PeWriter
                 }
 
                 nativePdbWriterOpt.WriteRemainingEmbeddedDocuments(mdWriter.Module.DebugDocumentsBuilder.EmbeddedDocuments);
+                nativePdbWriterOpt.WriteRemainingDebugDocuments(mdWriter.Module.DebugDocumentsBuilder.GetAllDebugDocuments());
```

Comment:
Nit: this boxes the `ImmutableArray<T>` into `IEnumerable<T>`. Given both these APIs are new consider changing the type of one to avoid the box.